src/Tizen.WebView/Tizen.WebView/BackForwardList.cs

```diff
@@ -0,0 +1,176 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Collections.Generic;
+
+namespace Tizen.WebView
+{
+    /// <summary>
+    /// This class provides the properties of Back Forward list item of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardListItem
+    {
+        private IntPtr _item_handle;
+
+        /// <summary>
+        /// Creates a Back Forward List Item object.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal BackForwardListItem(IntPtr handle)
+        {
+            _item_handle = handle;
+        }
+
+        /// <summary>
+        /// Url of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Url
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_url_get(_item_handle);
+            }
+        }
+
+        /// <summary>
+        /// Title of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Title
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_title_get(_item_handle);
+            }
+        }
+
+        /// <summary>
+        /// Original Url of the back forward list item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string OriginalUrl
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_back_forward_list_item_original_url_get(_item_handle);
+            }
+        }
+    }
+
+    /// <summary>
+    /// This class provides the properties of Back Forward list of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardList
+    {
+        private IntPtr _list_handle;
+
+        /// <summary>
+        /// Creates a Back Forward List object.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal BackForwardList(IntPtr handle)
+        {
+            _list_handle = handle;
+        }
+
+        /// <summary>
+        /// Current item of the back forward list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem CurrentItem
+        {
+            get
+            {
+                IntPtr ItemPtr = Interop.ChromiumEwk.ewk_back_forward_list_current_item_get(_list_handle);
```

Comment:
ItemPtr -> itemPtr
Plus, need null checking for it.

Reply:
Done